rest/data/perf.go

```diff
@@ -48,8 +48,7 @@ func (dbc *DBConnector) FindPerformanceResultsByTaskId(taskId string, interval u
 			TaskID: taskId,
 			Tags:   tags,
 		},
-		// set MaxDepth to -1 to avoid recursive child search
-		MaxDepth: -1,
```

Comment:
doesn't this revert a thing that you changed last week?

Reply:
No I forgot to change this with those changes (remember -1 now means to recurse without limit and 0 means do not recurse at all)